# Since FSL doesn't have a real configre, hardcoded changes are needed to compile GPU executables.
# Author: Caspar van Leeuwen (SURFsara)
diff -Nru fsl.orig/src/fdt/fslconfig fsl/src/fdt/fslconfig
--- fsl.orig/src/fdt/fslconfig	2019-10-11 13:50:36.336062960 +0200
+++ fsl/src/fdt/fslconfig	2019-10-11 18:09:38.260806371 +0200
@@ -1,3 +1 @@
-if [ `hostname` == "caper.fmrib.ox.ac.uk" -o `hostname` == "jalapeno19.fmrib.ox.ac.uk" ]; then
-     export MAKEOPTIONS="${MAKEOPTIONS} COMPILE_GPU=1" ;
-fi
+export MAKEOPTIONS="${MAKEOPTIONS} COMPILE_GPU=1" ;
diff -Nru fsl.orig/src/ptx2/fslconfig fsl/src/ptx2/fslconfig
--- fsl.orig/src/ptx2/fslconfig	2019-10-11 13:50:36.310062989 +0200
+++ fsl/src/ptx2/fslconfig	2019-10-11 18:09:19.874825110 +0200
@@ -1,3 +1 @@
-if [ `hostname` == "caper.fmrib.ox.ac.uk" -o `hostname` == "jalapeno19.fmrib.ox.ac.uk" ]; then
-     export MAKEOPTIONS="${MAKEOPTIONS} COMPILE_GPU=1" ;
-fi
+export MAKEOPTIONS="${MAKEOPTIONS} COMPILE_GPU=1" ;
